.github/workflows/prerelease_staged_publish.yml

```diff
@@ -73,6 +73,52 @@ jobs:
           gpg_passphrase: ${{ env.GPG_PASSPHRASE }}
           gpg_private_key_base64: ${{ env.GPG_PRIVATE_KEY_BASE64 }}
 
+  publishing-to-s3-macos:
+    name: Publish macos artifacts into s3 staging bucket
+    runs-on: ubuntu-20.04
+
+    strategy:
+      max-parallel: 1
+      matrix:
+        assetsType:
+          - "targz"
```

Comment:
we dont have deb, rpm here. also I think this wont work as this will clash with linux tar.gz 😞

Reply:
They shouldn't clash as in corruption, but both mac & linux tarballs will be released yes.

Although, we could split this as well https://github.com/newrelic/infrastructure-agent/pull/608